packages/paste-website/src/constants.ts

```diff
@@ -3,7 +3,8 @@ export const TWILIO_BLUE = '#0D122B';
 export const PSA_ALERT_HEIGHT = 40;
 export const PASTE_THEME_ALERT_HEIGHT = 54;
 export const SITE_MASTHEAD_HEIGHT = 76;
-export const SITE_BREAKPOINTS = ['768px', '1024px', '1220px'];
+// Added 1680 and above to account for really large monitors
```

Comment:
This comment is lying about the value added 🙈

Reply:
Forgot to update that one. I'll do that now.